capellambse/aird/parser/_edge_factories.py

```diff
@@ -115,6 +115,8 @@ def extract_bendpoints(
             ).attrib["id"]
         except (StopIteration, KeyError):
             sourceanchor = "(0.5, 0.5)"
+        if sourceanchor.endswith(" custom"):
+            sourceanchor = sourceanchor[: -len(" custom")]
```

Comment:
Do you have any clue/assumption what this custom is there for? I've found out that Capella uses [Draw2D](https://github.com/freegroup/draw2d) for their diagrams... but for the `geo.Point` type in Draw2D there is no custom suffix to be found. It seems that this is something specific in Capella.

Reply:
I presume it has to do with objects that were modified by hand (a.k.a. *custom*ized) in the Capella GUI, but that's ... just a theory. A code theory!